src/components/AddItem.js

```diff
@@ -19,61 +23,77 @@ const AddItemForm = props => {
 
   const addItem = e => {
     e.preventDefault();
-    console.log(emptyShoppingItem.lastPurchasedDate)
+
+    const items = shoppingList;
 
     if (enteredValue.name === '') {
       alert('Please enter an item name');
     } else {
-      firestore.collection('shoppingList').add({
-        name: enteredValue.name,
-        nextPurchase: parseInt(enteredValue.nextPurchase, 10),
-        token: localStorage.getItem('userToken'),
+      const removePunctuation = enteredValue.name.replace(
+        /(~|`|!|@|#|$|%|^|&|\*|\(|\)|{|}|\[|\]|;|:|"|'|<|,|\.|>|\?|\/|\\|\||-|_|\+|=)/g,
```

Comment:
Is there any way we can remove spacing here as well so that the final thing we are comparing is one solid string? I'd like it to alert me if I type in "cheesesticks" but I already have "Cheese sticks" on the list.